src/todos.js

```diff
@@ -0,0 +1,37 @@
+let express = require('express');
```

Comment:
1. Use `const` instead of `let`. This repository is setup with [Prettier](https://prettier.io/) and ESLint (https://eslint.org/), so this doesn't pass `yarn lint` (see [failing build](https://travis-ci.org/amercier/javascript-exercise/builds/466723495)).
2. It might be a good idea to consider switching to [koa](https://koajs.com/) (by the team behind Express), to leverage Node 8+ `async`/`await` powerful syntax.